vm/Cargo.toml

```diff
@@ -28,6 +28,9 @@ cairo-1-hints = [
 ]
 arbitrary = ["dep:arbitrary", "felt/arbitrary", "felt/std", "std"]
 lambdaworks-felt = ["felt/lambdaworks-felt"]
+# Allows extending the set of hints for the current vm run from within a hint.
+# For a usage example checkout vm/src/tests/run_deprecated_contract_class_simplified.rs
+extensive_hints = []
```

Comment:
Can we also add the feature to the cairo-vm-cli?
In `cairo-vm-cli/Cargo.toml`
`extensive_hints = ["cairo-vm/extensive_hints"]`